### DISCUSSION TITLE
Question about Connecting SIP Phone from external Network

### DISCUSSION THREAD

**Discussion by rolf-sobieralski** [sipsorcery-org/sipsorcery#812] [Q&A]
Hello Everybody,
i'm currently Building a VoIP Server to handle Incomming Calls from a SIPGate Account on my connected Devices.  Now my Question is, how do i Connect a Phone from a Parent, who is living some distance from My Home and has a Device connected to their Local Network.
i've already managed the SIP Package Transmission but it seams that there are some Dificulties with the Audio sending and Receiving. i've seen the Methods "SetDestination"," SetRemoteDescription" , "CreateAnswer" and "CreateOffer" but i can't find any good Explanationfor myself on this Methods. only Thing i know, these Methods have somethong to do with the SDP Information where to send the audio Packages. as far as i can tell the Addresses on the Media Sessions where the Packages are send to are solely comming from the SDP Answer of the Remote Device(the Device Connected to the Network of my Parent). so it would be Great, if someone could give me a short Explanation on this Methods or point me to the Right Direction.

Greetings
Rolf

**Comment by sipsorcery** [comment]
Hi Rolf,

I'm not exactly sure what your question is?

Do you want to be able to place a SIP call to your parent's device that is resitered with sipgate?

If so and it's a standard SIP call then the getting started example should be close to waht you need.

https://github.com/sipsorcery-org/sipsorcery/blob/master/examples/SIPExamples/GetStarted/Program.cs

Regards,
Aaron
